src/integration/java/io/pinecone/helpers/IndexManager.java

```diff
@@ -80,7 +81,7 @@ public static IndexModel waitUntilIndexIsReady(Pinecone pinecone, String indexNa
         int waitedTimeMs = 0;
         int intervalMs = 2000;
 
-        while (!index.getStatus().getReady()) {
+        while (index.getStatus().getState() != IndexModelStatus.StateEnum.READY) {
```

Comment:
Not impt, but I was just reading in Effective Java that it's better to have binary `Enum` values instead of booleans, so nice job w/the `StateEnum` guy!

Reply:
Thanks, but it's part of the generated OpenAPI code. 🫣